### DISCUSSION TITLE
Evaluate :what do we mean by evaluation data?

### DISCUSSION THREAD

**Discussion by Evarln** [explosion/spaCy#9929] [Help: Other Questions] [answered]
Hello,

I am reading the documentation to evaluate the named entity recognition model I trained via Spacy. The order is as follows: 
python -m spacy evaluate model data_path

But I don't understand what is meant by evaluation data. To me, the evaluation data is the test data that is annotated by my new model. However, to evaluate, we need a gold standard. So I don't really understand what data to put.

Thanks!!

**Comment by polm** [comment, marked as answer]
When you prepare annotated examples for your model, you should use some as training data, and hold some out as evaluation / validation / dev data. 

Please see [the docs](https://spacy.io/usage/training#:~:text=also%20need%20evaluation%20data.) or [the spaCy course](https://course.spacy.io/en/) for more details.
